feature-configs/demo/dpdk/build-config.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: s2i-dpdk-app
+  namespace: dpdk-testing
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
```

Comment:
after deploying these configs, is there a "wait" step that we can introduce that verifies the build is successful 

look at the `make feature-wait` logic

Reply:
Hi @davidvossel thanks for the comment can you take another look?

Reply:
I think this is being covered now.